master/templates/secrets/guardian.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: guardian
+type: Opaque
+data:
+  secret_key: bG9jYWxob3N0Cg== #localhost
```

Comment:
why is the "secret_key" localhost?  Shoudl this have a different name or what is it used for?